docker-compose.yml

```diff
@@ -34,7 +35,7 @@ services:
       start_period: 3s
 
   cockroach:
-    image: cockroachdb/cockroach:latest-v21.1
+    image: cockroachdb/cockroach:latest-v22.2
```

Comment:
Don't do this too. I don't think the minimum supported version is a part of this PR. If there is any issue with versions, please explain the reason in the PR description.

Reply:
Reverted.